ci/scripts/jenkins/post_benchmark.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+source ${WORKSPACE}/ci/scripts/jenkins/common.sh
+
+REPORTS_DIR="${WORKSPACE_TMP}/reports"
+
+gpuci_logger "Archiving benchmark reports"
+cd $(dirname ${REPORTS_DIR})
+tar cfj ${WORKSPACE_TMP}/benchmark_reports.tar.bz $(basename ${REPORTS_DIR})
+
+gpuci_logger "Pushing results to ${DISPLAY_ARTIFACT_URL}/"
+aws s3 cp ${WORKSPACE_TMP}/benchmark_reports.tar.bz "${ARTIFACT_URL}/benchmark_reports.tar.bz"
+
+exit $(cat ${WORKSPACE_TMP}/exit_status)
```

Comment:
This seems really odd to me. Is there no other way to pass information from one script to the next? Could we not have a `benchmark_wrapper.sh` script that does:
```bash
# Do pre-benchmark stuff here

# Run benchmark
${WORKSPACE}/ci/scripts/jenkins/benchmark.sh
BENCHMARK_EXIT_STATUS=$?

# Post benchmark stuff
```

Reply:
I don't think so not with the way its setup currently with the three scripts as separate steps.